Samples/blazor-add-in/excel-blazor-add-in/excel-blazor-add-in/wwwroot/BlazorAddIn.lib.module.js

```diff
@@ -0,0 +1,41 @@
+﻿/* Copyright(c) Maarten van Stam. All rights reserved. Licensed under the MIT License. */
+/**
+ * JavaScript Initializers
+ * JavaScript (JS) initializers execute logic before and after a Blazor app loads. JS initializers are useful in the following scenarios:
+ * - Customizing how a Blazor app loads.
+ * - Initializing libraries before Blazor starts up.
+ * - Configuring Blazor settings.
+ */
+
+/**
+ * beforeStart(options, extensions): 
+ * 
+ * Called before Blazor starts. 
+ * For example, beforeStart is used to customize the loading process, logging level, and other options specific to the hosting model. 
+ * @param  {} wasmoptions
+ * @param  {} extensions
+ */
+export async function beforeStart(wasmoptions, extensions) {
+    console.log("We are now entering function: beforeStart");
+
+    Office.onReady((info) => {
+        // Check that we loaded into Excel.
+        if (info.host === Office.HostType.Excel) {
+            console.log("We are now hosting in Excel.");
+        }
+        else {
+            console.log("We are now hosting in The Browser (of your choice).");
```

Comment:
@aafvstam It's unclear if this code is intended to check for running in Excel. Shouldn't it log something like "This add-in only runs correctly in Excel. Please load the add-in in Excel." I notice this same code is in the Word add-in so we should update that one as well.

Reply:
The code is not really checking, I use this to preload Office JS before Blazor kicks in (I ran into issues loading the taskpane when not preloading before Blazor). Besides that I think the host is determined by the manifest already: 
`<Host Name="Workbook" />`
It won't show up in other hosts if I'm correct.